controllers/articles.controllers.js

```diff
@@ -26,3 +28,12 @@ exports.patchArticleVotes = (req, res, next) => {
     .then((article) => res.status(200).send({ article }))
     .catch(next);
 };
+
+exports.getArticleComments = async (req, res, next) => {
+  const { article_id } = req.params;
+  const comments = await fetchArticleComments(article_id)
+    .then((comments) => {
```

Comment:
Try to be consistent. If you're going to use async/await, you probably should use it for the whole thing. Here, you don't need the `.then`, you can just `res.status(200).send({ comments });` because you already have access to the comments when you created the comments variable.